airbyte-integrations/connectors/source-prestashop/integration_tests/spec.json

```diff
@@ -4,16 +4,20 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I suggest we should remove this file in favour of passing a spec.yaml to a test config

Reply:
integration_tests/spec.json -> source_prestashop/spec.yaml